.github/workflows/main.yml

```diff
@@ -134,27 +154,30 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: set up python
-        uses: actions/setup-python@v5
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
         with:
-          python-version: "3.12"
+          version: "0.4.12"
+          enable-cache: true
+
+      - name: Set up Python 3.12
+        run: uv python install 3.12
 
       - name: check GITHUB_REF matches package version
         uses: samuelcolvin/check-python-version@v4.1
         with:
           version_file_path: pyproject.toml
 
-      - uses: eifinger/setup-rye@v2
-      - run: rye config --set-bool behavior.use-uv=true
-      - run: rye build
+      - run: uv build
 
       - name: Publish logfire to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Does this not likely use the `python` command or whatever? Does it work without `uv run`? Is there a way to dry run to test it?

Reply:
No. Also, the recommendation is to do this: https://github.com/Kludex/python-multipart/blob/5303590e767c8f6125729912923cddedb1cc0946/.github/workflows/publish.yml#L40-L59